src/RoleEnum.js

```diff
@@ -0,0 +1,9 @@
+const RoleEnum = {
```

Comment:
Let's put this in a new folder under /src called /Enums so we can start a collection!

Reply:
Good idea. Just updated the file structure.